view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -56,19 +58,120 @@ define(
 
+            /**
+             * Format Price
+             * 
+             * @param {float} amount - Amount to be formatted
+             * @return {string}
+             */
+            getFormattedAmount: function (amount) {
+                return priceUtils.formatPrice(amount, quote.getPriceFormat());
+            },
+
+            /**
+             * Get Installment minimum
+             * this function respects info from: https://www.omise.co/installment-payment
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             *
+             * @param {string} id - Bank ID
+             * @return {integer}
+             */
+            getInstallmentMinimum: function (id) {
+                switch (id) {
+                    case 'kbank': return 500;
+                    case 'bbl': return 500;
+                    case 'bay': return 300;
+                    case 'first_choice': return 300;
+                    case 'ktc': return 300;
+                }
+            },
+
+            /**
+             * Get Installment monthly interest rate
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             * 
+             * @param {string} id - Bank id
+             * @return {float}
+             */
+            getInstallmentInterestRate: function (id) {
+                switch (id) {
+                    case 'kbank': return 0.0065;
+                    case 'bbl': return 0.008;
+                    case 'bay': return 0.008;
+                    case 'first_choice': return 0.013;
+                    case 'ktc': return 0.008;
+                }
+            },
+
+            /**
+             * Get VAT tax rate
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             *
+             * @return {float}
+             */
+            getVatRate: function () {
+                return 0.07;
+            },
+
+            /**
+             * Get zero interest setting
+             *
+             * @return {string}
+             */
+            isZeroInterest: function () {
+                return window.checkoutConfig.is_zero_interest;
+            },
+
+            /**
+             * Calculates single installment amount
+             * 
+             * @param {string} id - Bank ID
+             * @param {integer} terms - number of monthly installments
+             * @return {integer}
+             */
+            calculateSingleInstallmentAmount: function (id, terms) {
+                const total = this.getTotal();
+                const tax = this.getVatRate();
+
+                if (this.isZeroInterest()) {
+                    //merchant pays interest
+                    return (total / terms).toFixed(0)
+                }
+
+                // get monthly interest setting
+                const rate = this.getInstallmentInterestRate(id);
```

Comment:
IMO, if a method or a function has been named properly, we wouldn't need to have a code comment here.
If we feel to comment it, maybe we should consider to rename the function.

Reply:
i removed that comments @guzzilar

Reply:
Thanks @jacstn